validator/node/node.go

```diff
@@ -821,15 +818,13 @@ func (c *ValidatorClient) registerRPCGatewayService(router *mux.Router) error {
 		),
 		gwruntime.WithForwardResponseOption(gateway.HttpResponseModifier),
 	)
-	muxHandler := func(apiMware *apimiddleware.ApiProxyMiddleware, h http.HandlerFunc, w http.ResponseWriter, req *http.Request) {
+
+	// TODO: remove api middleware from this
```

Comment:
I would remove this TODO because the code will fail to compile once we remove the middleware parameter from `WithMuxHandler`